openedx/openedx-platform
# Bump edx-val

This PR makes sure transcripts are imported even when the video already exists in the platform.

Check the related PR at https://github.com/edx/edx-val/pull/200

Temporarily using the repo+hash for testing purposes.

**Sandbox URL**: https://studio.pr22767.sandbox.opencraft.hosting/

**Testing instructions**:

1. Create a course, add a video and a transcript
1. Export the course
1. Remove the transcript from edxval using the Django admin at `/admin/edxval/videotranscript/`
1. Import the course and make sure the transcript is added to the video
